en/docs/install-and-setup/setup/setting-up-databases/overview.md

```diff
@@ -30,6 +30,16 @@ WSO2 products are shipped with scripts for creating the required tables in all t
 -   [Changing to IBM DB2]({{base_path}}/install-and-setup/setting-up-databases/changing-default-databases/changing-to-ibm-db2)
 -   [Changing to MariaDB]({{base_path}}/install-and-setup/setting-up-databases/changing-default-databases/changing-to-mariadb)
 
+
+!!! Note
+
+    - To enable the default H2 database please add below configurations to deployment.toml file in <APIM_HOME>/repository/conf and restart the server.
```

Comment:
'To enable the default H2 database, add the below configurations to deployment.toml file'

Reply:
Fix https://github.com/wso2/docs-apim/pull/2550/commits/3af9262137499afc9ab91481a7acb23857cddc80